lib/cn/vblock_builder.c

```diff
@@ -15,24 +18,106 @@
 #include <hse_util/logging.h>
 #include <hse_util/perfc.h>
 #include <hse_util/vlb.h>
+#include <hse_util/hse_err.h>
 
+#include <hse_ikvdb/blk_list.h>
+#include <hse_ikvdb/mclass_policy.h>
 #include <hse_ikvdb/tuple.h>
 #include <hse_ikvdb/limits.h>
 #include <hse_ikvdb/kvset_builder.h>
 #include <hse_ikvdb/cn.h>
 
 #include <hse/limits.h>
 #include <hse/kvdb_perfc.h>
+#include <mpool/mpool.h>
 
+#include "vblock_builder.h"
 #include "omf.h"
 #include "blk_list.h"
 #include "cn_mblocks.h"
 #include "cn_metrics.h"
 #include "cn_perfc.h"
 
-#include <mpool/mpool.h>
+#define WBUF_LEN_MAX      ((1024 * 1024) + VBLOCK_FOOTER_LEN)
```

Comment:
What is the significance of the `1024 * 1024`.

Reply:
1MiB IOs